solomon/client.c

```diff
@@ -1,41 +1,30 @@
-#include <arpa/inet.h>
-#include <stdlib.h>
-#include <stdio.h>
-#include <sys/types.h>
-#include <sys/socket.h>
-#include <netinet/in.h>
-#include <unistd.h>
-#include <dataStructDefault.h>
+#include <server.h>
 
-#define BUFFER_SIZE 1024
-#define PORT 8080
-
-#define ANSI_COLOR_RED     "\x1b[31m"
-#define ANSI_COLOR_GREEN   "\x1b[32m"
-#define ANSI_COLOR_YELLOW  "\x1b[33m"
-#define ANSI_COLOR_BLUE    "\x1b[34m"
-#define ANSI_COLOR_MAGENTA "\x1b[35m"
-#define ANSI_COLOR_CYAN    "\x1b[36m"
-#define ANSI_COLOR_RESET   "\x1b[0m"
-
-/*	Client Socket Workflow
-    1. Socket;
-    2. Connect;
-    3. Send/Receive Message;
-    4. Close Connection.
+/*
+	Client Socket Workflow
+	1. Socket;
+	2. Connect;
+	3. Send/Receive Message;
+	4. Close Connection.
 */
 
 int main(int argc, char * const argv[]){
     if(argc != 3) {
```

Comment:
"Erradicate those magic numbers!!"
To brincando, só se vc não se sentir incomodado.  :stuck_out_tongue:

Reply:
Hahaha não me responsabilizo por esses números!!!